### ISSUE TITLE
v6.70 Source Code is already shared.

### ISSUE THREAD

**Issue by dkcommunity** [opendarkeden/server#124]
Guys, leave this old useless pile of code behind, I shared the client and server source codes of the 6.70 version in my discord, it is useful to focus on it. You can click on my username for information.

**Comment by tiancaiamao** [comment]
I hope there would be an 'active' project, and be open-source, so people can really work together on it.
A closed-source code is a dead project, if you want to create a real community which people can collaborate, it's better to put the code on the github.

@dkcommunity

**Comment by Supacek** [comment]
we like V2 opendarkeden 🤣

**Comment by dkcommunity** [comment]
Yes, I totally agree with you on this. But it looks like you're doing this job better, I already have a discord server I'm interested in and I don't have room for an extra project. I leave this job to you.
If you want to access the files, I invite you to our discord server.

@tiancaiamao
